platform/mellanox/peripheral_table.j2

```diff
@@ -0,0 +1,20 @@
+{%- if DEVICE_METADATA is defined and DEVICE_METADATA['localhost']['platform'] is defined %}
+{%-     set platform = DEVICE_METADATA['localhost']['platform'] %}
+{%- else -%}
+{%-     set platform = "vs-platform" %}
```

Comment:
The same issue, move `vs` related part into `platform/vs/peripheral_table.j2`. Please don't mix platforms into `mellanox` folder.